morf-integration-test/pom.xml

```diff
@@ -30,6 +34,19 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-source-plugin</artifactId>
       </plugin>
+      <plugin>

```

Comment:
So because this runs last (depending upon the others), we can safely aggregate here in ``verify`` :+1: